skel/.bashrc

```diff
@@ -0,0 +1,10 @@
+## Template user .bashrc
```

Comment:
The default skel/.bashrc have a lot of definitions that the would lose if this file is overwritten. I wonder if the definitions in this file will be appended to the default one or overwritten

Reply:
Yeah looking into things, it might be best to put these into /etc/bashrc. Then we can keep the zsh install script we currently have that sources /etc/profile, but instead point it to bashrc since the two are fairly compatible. Fish isn't POSIX compliant or bash-like so unfortunately will require a different approach.

Reply:
Okay right, the plan is:
Put current skel/.bashrc code into bashrc. Then on install of zsh we can /etc/zsh/zprofile to source /etc/profile, and /etc/zsh/zshrc to source /etc/bashrc.

@sirredbeard what do you think? Choices are either: undo this merge, or make further commits removing .bashrc and .zshrc from skel but leave skel in place as I have a feeling we'll need it for handling Fish shell.

Reply:
I trust your judgment on this one.

Reply:
Based on risk of messing up the Git work tree I'm going to make all changes as new commits from here as opposed to undoing any.